buckbuild.bzl

```diff
@@ -1909,7 +1943,146 @@ def define_buck_targets(
             C10,
         ],
     )
+    # Note : following aten_quantized_linear library is manually designed to identify all requires symbols to create a shared library
+    # This library is strictly required to add a feature quantized gradient function with backend kernel
+    # Because custom gradient function calls backend kernel matrix multiplication for gradient computation for memory efficiency
+    # ,which is located in aten/src/ATen/native/quantized/cpu/qlinear_prepack.cpp
+    # Refer - D42593235
+    aten_quantized_linear = [
```

Comment:
Why does this list need to be explicit here? I would expect we already have such a list somewhere?